sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/AvoidExcessiveInheritance_CustomValues.cs

```diff
@@ -3,8 +3,12 @@
 
 namespace Tests.Diagnostics
 {
-    class FirstClass : Exception { }

```

Comment:
This is just a rename. I find it easier to read Class_0 (class with 0 base classes in the same namespace) than FirstSubclass.